app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -408,6 +408,13 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
                 }
             }
         })
+
+        //scroll recycler view up when keyboard shows up
+        recycler_view.addOnLayoutChangeListener { v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom ->
+            if (bottom < oldBottom) {
+                recycler_view.smoothScrollToPosition(0)
+            }
+        }
```

Comment:
Can you please remove this? We've already solved this keyboard issue on another PR. :)